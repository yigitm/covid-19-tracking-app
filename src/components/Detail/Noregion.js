import React from 'react';
import { Card, Col } from 'react-bootstrap';
import Loading from '../Home/Loading';

const noRegion = () => (
  <Col className="col-12 bg-red">
    <Loading />
    <Card.Body className="d-flex justify-content-between">
      <i className="fas fa-chart-pie text-left text-light icon-header" />

      <Card.Title className="text-center font-big">
        <span className="font-region text-light">No Regions</span>
      </Card.Title>
    </Card.Body>
  </Col>
);
export default noRegion;
