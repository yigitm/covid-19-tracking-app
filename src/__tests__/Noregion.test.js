import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { render } from '@testing-library/react';
import Noregion from '../components/Detail/Noregion';

describe('Test Noregion render', () => {
  test('Test Noregion', () => {
    const tree = render(
      <BrowserRouter>
        <Noregion />
      </BrowserRouter>,
    );
    expect(tree).toMatchSnapshot();
  });
});
