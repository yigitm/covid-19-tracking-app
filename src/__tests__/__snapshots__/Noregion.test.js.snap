// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Noregion render Test Noregion 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="col-12 bg-red col"
      >
        <div
          class="w-100 h-100 text-light loading bg-red"
        >
          <div
            class="d-flex justify-content-center align-items-center"
          >
            <div
              class="spinner-grow spinner-grow-md"
            />
            <div
              class="spinner-grow spinner-grow-md"
            />
            <div
              class="spinner-grow spinner-grow-md"
            />
            <h1
              class=""
            >
              Loading...
            </h1>
          </div>
        </div>
        <div
          class="d-flex justify-content-between card-body"
        >
          <i
            class="fas fa-chart-pie text-left text-light icon-header"
          />
          <div
            class="text-center font-big card-title h5"
          >
            <span
              class="font-region text-light"
            >
              No Regions
            </span>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="col-12 bg-red col"
    >
      <div
        class="w-100 h-100 text-light loading bg-red"
      >
        <div
          class="d-flex justify-content-center align-items-center"
        >
          <div
            class="spinner-grow spinner-grow-md"
          />
          <div
            class="spinner-grow spinner-grow-md"
          />
          <div
            class="spinner-grow spinner-grow-md"
          />
          <h1
            class=""
          >
            Loading...
          </h1>
        </div>
      </div>
      <div
        class="d-flex justify-content-between card-body"
      >
        <i
          class="fas fa-chart-pie text-left text-light icon-header"
        />
        <div
          class="text-center font-big card-title h5"
        >
          <span
            class="font-region text-light"
          >
            No Regions
          </span>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
